addinstanceform.php

```diff
@@ -20,11 +20,14 @@
  * @package   cachestore_redis
  * @copyright 2013 Adam Durana
  * @license   http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ * @var $CFG  stdClass
  */
 
 defined('MOODLE_INTERNAL') || die();
 
+global $CFG;
```

Comment:
Hi @dmitriim ... this was never used until I required that test from a testcase, which did not have $CFG declared in the same scope and the following line (require_once) would fail because $CFG is not defined.